packages/components/psammead-most-read/src/Item/index.jsx

```diff
@@ -48,7 +50,10 @@ const StyledLink = styled.a`
   }
 `;
 
+const alignRankToLinkSpacing = '0.375rem';
```

Comment:
Just for consistency:

```suggestion
const ALIGN_RANK_TO_LINK_SPACING = '0.375rem';
```